### PR TITLE
Truncate title names to fix very long name issue

### PR DESCRIPTION
**Pull request by cedryc**
create -l | --max_title_length option

### CODE REVIEW

File: i3_workspace_names_daemon.py [hunk 1]
```diff
@@ -22,7 +22,7 @@
 }
 
 
-def build_rename(i3, app_icons, delim, uniq):
+def build_rename(i3, app_icons, delim, lenght, uniq):
```
Comment (cboddy) [inline i3_workspace_names_daemon.py:25]: mind fixing the typo?

Comment (cboddy) [reply i3_workspace_names_daemon.py:25]: and in the other places, please.

**Review by cboddy — CHANGES_REQUESTED** [review]

**Comment by cboddy** [comment]
One minor comment, otherwise LGTM!

**Comment by cboddy** [comment]
@cedl38  thanks for this. I've fixed the typos and a bug that caused the icon lookup to fail - see [5d37153f](chttps://github.com/cboddy/i3-workspace-names-daemon/commit/5d37153fcc4bde5c8b800be1dd6cb90d9426d8c8).

This is now pushed to pypi in version ``0.13.0``.

**Comment by cedryc** [comment]
> @cedl38 thanks for this. I've fixed the typos and a bug that caused the icon lookup to fail - see [5d37153](https://github.com/cboddy/i3-workspace-names-daemon/commit/5d37153fcc4bde5c8b800be1dd6cb90d9426d8c8).
> 
> This is now pushed to pypi in version `0.13.0`.

Sorry I forget to make requested changes. Thanks for fixing it :)

**Pull request merged**